src/Tracing/Sampler/AlwaysOffSampler.php

```diff
@@ -15,8 +16,13 @@ class AlwaysOffSampler implements SamplerInterface
      *
      * @return bool
      */
-    public function shouldSample()
+    public function shouldSample(): bool
```

Comment:
Gah I completely missed this return and the AlwaysOn one too.  Thanks for the update!

Reply:
Tests are failing because of SpanContext::export method missing. It was remove in [6c9b99cfcdbf3d5ef4d4efc1595f6b7e7f5a1d80](https://github.com/open-telemetry/opentelemetry-php/commit/6c9b99cfcdbf3d5ef4d4efc1595f6b7e7f5a1d80#diff-b0672e6dc8ef53bdc61451969c5ecfe0L26). I don't know the reason it was removed. Should I change tests to pass without this method or should I add the method back in SpanContext?

Reply:
I'd say add it back in! Thank you!

Reply:
Added exporter method back and fixed tests and phan issues